openrtb_ext/imp_mobfoxpb.go

```diff
@@ -0,0 +1,4 @@
+package openrtb_ext
+
+// ExtImpMobfoxpb defines mobfox specifiec param
```

Comment:
Can you please remove this struct definition as it is not being used anywhere?

Reply:
done